sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/util/SyncBlobHelper.java

```diff
@@ -0,0 +1,38 @@
+package com.azure.storage.blob.implementation.util;
+
+import com.azure.storage.blob.implementation.AzureBlobStorageImpl;
+import com.azure.storage.blob.models.CpkInfo;
+import com.azure.storage.blob.specialized.BlobClientBase;
+
+public class SyncBlobHelper {
```

Comment:
Why does this have less functionality than the `BlobHelper`? Additionally, does this class need to exist as we could call into `BlobHelper` everywhere we call into this as all sync clients will have a corresponding reference to their async client?

Reply:
Does this get used anywhere?

Reply:
No, originally thinking need a helper for sync client. Go through the code, no need to have it for current version.